### DISCUSSION TITLE
SSL aktivieren auf der Solectrus WebApp

### DISCUSSION THREAD

**Discussion by markusklein01** [solectrus/solectrus#2422] [General]
Wie kann ich SSL auf der Solectrus WebApp aktivieren?

**Comment by ledermann** [comment]
Aktivieren im Sinne von "einfach einschalten" ist leider nicht vorgesehen. Für SSL benötigt man ein Zertifikat, das ganze Handling dazu lagert man besser an einen vorgeschalteten Reverse-Proxy aus, z.B. Traefik. Ich handhabe das in der [Live-Demo](https://solectrus.ledermann.dev) so, die Konfiguration ist aber nicht ganz trivial.

Mehr zum Thema hier:
https://github.com/solectrus/hosting/issues/21
